packages/client/src/i18n/nl.ts

```diff
@@ -77,18 +83,80 @@ export default {
           "het plaatsen van de zonwering, het rolhek, rolluik of luik zelf.",
       },
     },
+    question: {
+      alert: {
+        "this answer causes a need for permit":
+          "Door dit antwoord hebt u een vergunning nodig.",
+        "if you make another choice you might not need a permit":
+          "Als u een andere keuze maakt, hebt u misschien geen vergunning nodig.",
+        "this anwser makes it unable to determine the outcome":
+          "Door dit antwoord kunnen we niet vaststellen of u een vergunning nodig hebt.",
+      },
+    },
+    // @TODO: remove this, because this makes no sense!
```

Comment:
If it makes no sense, please remove it. Otherwise provide a meaningful comment. Thanks